styles/mobile.css

```diff
@@ -30,6 +30,9 @@
         margin-bottom: 1em;
         height: 0;
         overflow: hidden;
+
+        transition-duration: 500ms;
+        transition-timing-function: ease-in-out;
```

Comment:
Put transition declarations inside the rule for the element in its default state.  Otherwise the animation only occurs when the element goes from its 'changed' state back to its default state.